angular/src/app/interfaces/league-details.ts

```diff
@@ -7,7 +7,7 @@ export interface LeagueDetails {
   "image_source": string,
   "league_admin": User,
   "league_players": User[],
-  "number_of_finished_rounds": number,
-  "number_of_in_progress_rounds": number,
-  "number_of_rounds": number,
+  "numberOfFinishedRounds": number,
+  "numberOfInProgressRounds": number,
+  "numberOfRounds": number,
```

Comment:
if these fields are camelCase, what about image_source, league_admin, and so on?

Reply:
fixed